src/helpers/directives.ts

```diff
@@ -24,3 +57,11 @@ export function* loop<T>(
     }
   }
 }
+
+/**
+ * @description
+ * Repeats a template a certain number of times
+ */
+export function repeatCount(repetitions: number, template: TimesReturn): TemplateResult {
```

Comment:
nit naming: `Count` seems redundant, suggest just `repeat`

Reply:
I agree. However, `repeat` is a built-in Lit directive

<https://lit.dev/docs/templates/directives/#repeat>

The Lit `repeat` directive is more or less an optimized `map` function (so doesn't satisfy this directives use case).

I'd like to not have conflicting directive names (because it makes importing them confusing).

I'm VERY open to alternative naming suggestions

Reply:
rails uses `5.times` a lot... but just using `times(5, ...)` would not read well

Did you see this from lit: https://lit.dev/docs/templates/directives/#range?

Reply:
I didn't see `range`, I think that's exactly what I was looking for